### PR TITLE
added Moshi and HikariJSON

### PR DESCRIPTION
**Pull request by Moshe-Immerman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -450,13 +450,15 @@ A curated list of awesome Java frameworks, libraries and software.
...
 
+* [HikariJSON](https://github.com/brettwooldridge/HikariJSON) - High-performance JSON parser.
 * [Genson](http://owlike.github.io/genson/) - Powerful and easy to use Java to JSON conversion library.
 * [Gson](https://github.com/google/gson) - Serializes objects to JSON and vice versa. Good performance with on-the-fly usage.
 * [Jackson-datatype-money](https://github.com/zalando/jackson-datatype-money) - Open-source Jackson module to support JSON serialization and deserialization of JavaMoney data types.
 * [Jackson](http://wiki.fasterxml.com/JacksonHome) - Similar to GSON but has performance gains if you need to instantiate the library more often.
 * [JSON-io](https://github.com/jdereg/json-io) - Convert Java to JSON. Convert JSON to Java. Pretty print JSON. Java JSON serializer.
 * [jsoniter](http://jsoniter.com) - Fast and flexible library with iterator and lazy parsing API.
 * [LoganSquare](https://github.com/bluelinelabs/LoganSquare) - JSON parsing and serializing library based on Jackson's streaming API. Outperforms GSON & Jackson's library.
+* [Moshi](https://github.com/square/moshi) - A modern JSON library for Android and Java.
```
Comment (akullpp) [inline README.md]: If an entry has a similar scope as other entries in the same category, the description must state the unique features that distinguishes it from the other entries.

File: README.md [hunk 2]
```diff
@@ -450,13 +450,15 @@ A curated list of awesome Java frameworks, libraries and software.
 
 *Libraries for serializing and deserializing JSON to and from Java objects.*
 
+* [HikariJSON](https://github.com/brettwooldridge/HikariJSON) - High-performance JSON parser.
```
Comment (akullpp) [inline README.md]: See below.

Comment (Moshe-Immerman) [inline README.md]: @akullpp please see updated descriptions

File: README.md [hunk 3]
```diff
@@ -450,13 +450,15 @@ A curated list of awesome Java frameworks, libraries and software.
 
 *Libraries for serializing and deserializing JSON to and from Java objects.*
 
+* [HikariJSON](https://github.com/brettwooldridge/HikariJSON) - High-performance JSON parser, 2x faster than Jackson.
```
Comment (akullpp) [inline README.md]: Last update 2 years ago?

**Comment by akullpp** [comment]
If I have learned one thing curating this list, it's that if you have no idea what to do, you write a JSON parser. Everybody claims they are the fastest...

**Pull request merged**